modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/UninitializedOperationMessageProcessor.java

```diff
@@ -0,0 +1,266 @@
+          null,
+          operationMessageProcessorDelegate.retryPolicyTemplate,
+          muleContext.getExtensionManager(),
+          policyManager,
+          registry.lookupByType(ReflectionCache.class).get());
+
+    this.operationMessageProcessorDelegate = operationMessageProcessorDelegate;
+    this.resolvedParameters = resolvedParameters;
+  }
+
+  @Override
+  protected void doInitialise() throws InitialisationException {
+    //Do nothing
+  }
+
+  @Override
+  public ReturnDelegate createReturnDelegate() {
+    return operationMessageProcessorDelegate.createReturnDelegate();
+  }
+
+  @Override
+  public TargetReturnDelegate getTargetReturnDelegate() {
+    return operationMessageProcessorDelegate.getTargetReturnDelegate();
+  }
+
+  @Override
+  public ValueReturnDelegate getValueReturnDelegate() {
+    return operationMessageProcessorDelegate.getValueReturnDelegate();
+  }
+
+  @Override
+  public boolean isTargetPresent() {
+    return operationMessageProcessorDelegate.isTargetPresent();
+  }
+
+  @Override
+  public Optional<String> getTarget() {
+    return operationMessageProcessorDelegate.getTarget();
+  }
+
+  @Override
+  public void doStart() throws MuleException {
+    operationMessageProcessorDelegate.doStart();
+  }
+
+  @Override
+  public void doStop() throws MuleException {
+    operationMessageProcessorDelegate.doStop();
+  }
+
+  @Override
+  public void doDispose() {
+    operationMessageProcessorDelegate.doDispose();
+  }
+
+  @Override
+  public ExecutionMediator createExecutionMediator() {
+    return operationMessageProcessorDelegate.createExecutionMediator();
+  }
+
+  @Override
+  public ParameterValueResolver getParameterValueResolver() {
+    return operationMessageProcessorDelegate.getParameterValueResolver();
+  }
+
+  @Override
+  public void resolveParameters(CoreEvent.Builder eventBuilder,
+                                BiConsumer<Map<String, Supplier<Object>>, ExecutionContext> afterConfigurer)
+      throws MuleException {
+    operationMessageProcessorDelegate.resolveParameters(eventBuilder, afterConfigurer);
+  }
+
+  @Override
+  public void disposeResolvedParameters(ExecutionContext<OperationModel> executionContext) {
+    operationMessageProcessorDelegate.disposeResolvedParameters(executionContext);
+  }
+
+
+  @Override
+  protected Map<String, Object> getResolutionResult(CoreEvent event, Optional<ConfigurationInstance> configuration)
+      throws MuleException {
+
+    ValueResolvingContext valueResolvingContext = ValueResolvingContext.builder(event).build();
+    Object config = resolverSet.resolve(valueResolvingContext).asMap().get("config-ref");
+
+    Map resolvedParametersMap = resolvedParameters.entrySet().stream()
+        .filter(e -> e.getKey().equals("config-ref"))
+        .collect(Collectors.toMap(key -> key, value -> {
+          try {
+            return value.getValue().resolve(valueResolvingContext);
+          } catch (MuleException e) {
+            e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1148))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7719&issues=AWk7fTH-otIXnmsUoDr-&open=AWk7fTH-otIXnmsUoDr-)

[ISSUE_KEY]: AWk7fTH-otIXnmsUoDr-